src/Eppie.App/Eppie.App.ViewModels/MainPageViewModel.cs

```diff
@@ -446,7 +446,8 @@ public override async void OnNavigatedTo(object data)
 
                 await Core.RestoreFromBackupIfNeededAsync(new Uri(downloadUrl)).ConfigureAwait(true);
 
-                await RequestReviewAsync().ConfigureAwait(true);
+                // Startup auto-popups: show at most one per run
+                await TryShowStartupPopupOnceAsync().ConfigureAwait(true);
```

Comment:
I suggest renaming this function to one of the following:
- `ShowOneStartupPopupIfNeededAsync`
- `ShowOneStartupPopupIfApplicableAsync`
- `ShowOneStartupPopupIfAllowedAsync`